ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml

```diff
@@ -44,6 +44,7 @@ tests:
   commands: /usr/local/bin/cincinnati-graph-data
   container:
     from: graphdata-unittests
+  run_if_changed: internal-channels/candidate.yaml
```

Comment:
Also if the Rust unit-test code changes, so `run_if_changed: internal-channels/candidate.yaml|graph-data.rs/.*` or some such.